suite_cam4.xml

```diff
@@ -0,0 +1,204 @@
+      <scheme>cldfrc</scheme>                         <!-- Calculate cloud fraction -->
+      <scheme>cldfrc_rh2</scheme>                     <!-- Calculate cloud fraction with perturbed RH (is there a better way to do this, e.g. subcycle?) -->
+      <scheme>rhdfda_calc</scheme>                    <!-- Calculate "rhdfda", or the derivative d(RH)/da -->
+      <scheme>cldfrc_fice</scheme>                    <!-- Calculate ice cloud fraction -->
+      <scheme>strat_cond_repartition</scheme>         <!-- Re-partition stratiform cloud ice and cloud liquid, and calculate relevant heating rates -->
+      <scheme>pcond</scheme>                          <!-- Actual RK macro/micro-physics scheme -->
+      <scheme>rk_pcond_cam_update</scheme>            <!-- Placeholder for CAM ptend and diagnostic output calculations -->
+      <scheme>cldfrc</scheme>                         <!-- Calculate cloud fraction -->
+      <scheme>cldfrc_diag_output</scheme>             <!-- Calculate cloud fraction diagnostics and output them to history file -->
+      <scheme>cldefr</scheme>                         <!-- Calculate cloud liquid and ice particle effective radii, and output to history file/host model -->
+      <!-- **************************** -->
+      <scheme>convtran</scheme>                       <!-- Actual ZM convective transport scheme, for constituents in cnst_is_convtran2 -->
+      <scheme>check_tracers_chng</scheme>             <!-- Global tracer mass checker that kills model if error is large enough. -->
+      <scheme>diag_phys_writeout</scheme>             <!-- Calculate generic atmospheric physics diagnostics and output them to history file -->
+      <scheme>diag_conv</scheme>                      <!-- Calculate generic precipitation and convection diagnostics and output them to history file -->
+      <scheme>cloud_diagnostics_calc</scheme>         <!-- Calculate generic cloud diagnostics and output them to history file -->
+      <!-- CAM-RT radiation scheme -->
+      <!-- **************************** -->
+      <scheme>calc_solar_zenith_ang</scheme>          <!-- Calculate the solar zenith angle for the given time step -->
+      <scheme>group_day_night</scheme>                <!-- Determine which columns are in the day-side, vs night-side -->
+      <scheme>calc_col_mean_co2</scheme>              <!-- Calculate column mean CO2 values -->
+      <scheme>eccf_calc</scheme>                      <!-- Calculate "earth sun distance factor". -->
+      <scheme>tropopause_find</scheme>                <!-- Calculate tropopause height -->
+      <scheme>calc_col_mean_o2</scheme>               <!-- Calculate column mean O2 values -->
+      <scheme>aer_rad_props_sw</scheme>               <!-- Gather/calculate aerosol shortwave optical properties (potentially provided by registry/framework?) -->
```

Comment:
This type of thing is not currently on the radar. We have discussed the issue in an RRTMG interoperability meeting but have not dug into the 'data prep' details yet.

Reply:
Sounds good, I've gone ahead and removed the question from the comments, at least for now.